### PR TITLE
FHAC-877: Replace old COREX12 RealTime package names with newer ones.

### PR DESCRIPTION
**Pull request by danielrf123**
Replace old COREX12 RealTime package names with newer ones in CORE_X12DSRealTimeProxyConfig configuration file.
@mhpnguyen @sailajaa please review.

### CODE REVIEW

File: PerformanceTestTools/src/main/resources/CORE_X12DSRealTimeProxyConfig.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
     <!-- Java Implementation -->
     <bean id="adaptercore_x12dsrealtimejava"
           name="adaptercore_x12dsrealtimejava"
-          class="gov.hhs.fha.nhinc.corex12.docsubmission.realtime.adapter.proxy.AdapterCORE_X12DSRealTimeProxyJavaImpl"
+          class="gov.hhs.fha.nhinc.corex12.ds.realtime.adapter.proxy.AdapterCORE_X12DSRealTimeProxyJavaImpl"
```
Comment (mhpnguyen) [inline PerformanceTestTools/src/main/resources/CORE_X12DSRealTimeProxyConfig.xml]: Can you confirm that we have this class?

Comment (danielrf123) [inline PerformanceTestTools/src/main/resources/CORE_X12DSRealTimeProxyConfig.xml]: No!  That will be corrected.

**Comment by mhpnguyen** [comment]
Endorse for merging without any comments

**Comment by sailajaa** [comment]
I reviewed and good with PR merge.

**Pull request merged**